lerna.json

```diff
@@ -1,12 +1,17 @@
-{
-  "lerna": "2.8.0",
-  "packages": [
-    "packages/*"
-  ],
-  "command": {
-    "bootstrap": {
-      "npmClientArgs": ["--no-package-lock"]
-    }
-  },
-  "version": "independent"
-}
+{
+  "lerna": "2.8.0",
+  "packages": [
+    "packages/*"
+  ],
+  "command": {
+    "publish": {
+      "message": "chore(release): publish",

```

Comment:
This key should be defined for command version, since we will be using the version command locally and the publish command will be used in CI only.

Reply:
Now, the key "message" defines in command version.